delivery_carrier_label_seur/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+from . import model
```

Comment:
Could you rename it to models ? https://github.com/OCA/maintainer-tools/tree/master/template/module/models